### PR TITLE
fix: speed up template command by removing added sleep [CFG-1184]

### PR DESCRIPTION
**Pull request by teodora-sandu**
### What this does

This PR is a usability improvement. We are removing the "progress" logic which was printing out to the terminal the progress of templating a rule. For the progress logic to work, we needed to add some extra sleeping in the logic, which means customers have to wait longer than necessary for the command to finish. The output from the progress bar was going to the `stderr`, which is why the tests have been adapted accordingly.

It also includes a fix for the contract tests, where the `sed` command is slightly different based on the OS:  https://stackoverflow.com/questions/43171648/sed-gives-sed-cant-read-no-such-file-or-directory (I'm not sure why this worked before)

### Additional information
- [Jira ticket CFG-1184](https://snyksec.atlassian.net/browse/CFG-1184)
- [Slack thread](https://snyk.slack.com/archives/C022H54L8PJ/p1636477124142600)

### CODE REVIEW

File: spec/e2e/template_spec.sh [hunk 1]
```diff
@@ -22,13 +22,13 @@ Describe './snyk-iac-rules template ./fixtures/custom-rules --rule test'
...
       When call ./snyk-iac-rules template ./fixtures/custom-rules --rule test
       The status should be success
-      The stderr should include 'Template rules directory'
-      The stderr should include 'Template rules/test directory'
-      The stderr should include 'Template rules/test/main.rego file'
-      The stderr should include 'Template rules/test/main_test.rego file'
-      The stderr should include 'Template rules/test/fixtures directory'
-      The stderr should include 'Template rules/test/fixtures/allowed.tf file'
-      The stderr should include 'Template rules/test/fixtures/denied.tf file'
+      The output should include 'Template rules directory'
```
Comment (teodora-sandu) [inline spec/e2e/template_spec.sh:25]: The `stderr` was from the progress bar which was writing to the error stream. Now that we've deleted it, we don't need to read the errors

File: spec/contract/snyk_spec.sh [hunk 2]
```diff
@@ -51,7 +51,11 @@ Describe 'Contract test between the SDK and the Snyk CLI'
                 ./snyk-iac-rules template ./tmp --rule Contract
 
                 # replace the fixture path so it's correct
-                sed -i '' -e 's#/rules/Contract/fixtures#/tmp/rules/Contract/fixtures#' ./tmp/rules/Contract/main_test.rego
+                if [[ "$OSTYPE" == "darwin"* ]]; then
```
Comment (teodora-sandu) [inline spec/contract/snyk_spec.sh:54]: I'm not sure how this worked before ... https://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js

Comment (teodora-sandu) [inline spec/contract/snyk_spec.sh:54]: I'm not sure how this worked before ... https://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js

**Review by ipapast — APPROVED** [review]
Nice, tested it locally, we get instant output. ✅

**Pull request merged**